scripts/db_migrator.py

```diff
@@ -473,7 +473,22 @@ def version_2_0_0(self):
         Current latest version. Nothing to do here.
         """
         log.log_info('Handling version_2_0_0')
+        warmreboot_state = self.stateDB.get(self.stateDB.STATE_DB, 'WARM_RESTART_ENABLE_TABLE|system', 'enable')
+
+        if warmreboot_state != 'true':
+            portchannel_table = self.configDB.get_table('PORTCHANNEL')
+            for name, data in portchannel_table.items():
+                data['lacp_key'] = 'auto'
+                self.configDB.set_entry('PORTCHANNEL', name, data)
+            self.set_version('version_2_0_1')
+
+        return 'version_2_0_1'
 
+    def version_2_0_1(self):
```

Comment:
Adding @yxieca too to take a look at this change/approach to increment version in case of db_migrator. Thanks Ying.